code/modules/awaymissions/corpse.dm

```diff
@@ -450,7 +450,7 @@
 /obj/effect/mob_spawn/human/nanotrasensoldier
 	name = "Nanotrasen Private Security Officer"
 	uniform = /obj/item/clothing/under/rank/security
-	suit = /obj/item/clothing/suit/armor/vest
+	suit = /obj/item/clothing/suit/armor/swat
```

Comment:
literally what are you talking about? its just another variant of the other corpse in a different file